recruiter/templates/recruiter/recruiter_created_successfully.html

```diff
@@ -1,6 +1,6 @@
 {% extends 'base.html' %} {% block content %}
 <h3>
   You have successfully created you'r Recruiter account!
-  <a href="/login">let's login</a>
```

Comment:
I don't think this change is really related here, we can create a separate PR to change this text in the future

Reply:
That was wrong text and that's small u don't think every text change need a different PR. That's just make a tons of PR's and no one had time to review them.

Reply:
Personally, I don't really think that but I'm trying to follow the same train of thought of our mentors and this seems to be one of the guidelines.

Reply:
ok